pypdf/_merger.py

```diff
@@ -522,13 +514,13 @@ def _write_dests(self) -> None:
             raise RuntimeError(ERR_CLOSED_WRITER)
         for named_dest in self.named_dests:
             page_index = None
-            if "/Page" in named_dest:
+            if "/Page" in named_dest:  # deprecated
```

Comment:
Why did you add those comments?

Reply:
I've checked in coverage which lines where not covered, with the objective to get 100% for this file